### DISCUSSION TITLE
How to search in Select component on the label?

### DISCUSSION THREAD

**Discussion by DanishOnGit** [ant-design/ant-design#40828] [Q&A] [answered]
I am using `<Select mode='tags' options=[{label:galileo,value:1fef3322},{label:newton,value:jfierfr9i4r049}]/>` .

Now, when I try to search `gal` , galileo will not show up. But if I search `1fef` then galileo shows up!! 

**Expected behaviour:**
when I try to search `gal` , galileo should show up.

Please help!

**Comment by kiner-tang** [comment, marked as answer]
you can use like this
```tsx
<Select optionFilterProp="label" style={{width: 200}} mode='tags' options={[{label:'galileo',value:'1fef3322'},{label:'newton',value:'jfierfr9i4r049'}]}/> 
```

**Reply by DanishOnGit** [reply]
@kiner-tang  Thank you sooooo much!! This solves my problem and simplifies a lot of code for me!

**Reply by WatfaU** [reply]
Thank you so much @kiner-tang , this saved me a lot of time!

**Reply by pixelnx68** [reply]
Thank you @kiner-tang

**Comment by zombieJ** [comment]
Both `optionFilterProp` or `filterOption` can satisfy your requirement.
